scripts/lib/CIME/build.py

```diff
@@ -233,7 +233,7 @@ def _build_libraries(case, exeroot, sharedpath, caseroot, cimeroot, libroot, lid
     if mpilib == "mpi-serial":
         libs.insert(0, mpilib)
 
-    if cam_target == "preqx_kokkos":
+    if cam_target in ("preqx_kokkos", "theta-l"):
```

Comment:
Can we combine this and line 499 so that it only appears in one place?

Reply:
Good idea, will do